src/Intouch/LaravelNewrelic/LaravelNewrelicServiceProvider.php

```diff
@@ -98,7 +100,7 @@ function ( $request, $response ) use ( $me, $app )
      */
     public function getTransactionName( $request, $response, $app )
     {
-        $nameProvider = $app['config']['laravel-newrelic::name_provider'];
+        $nameProvider = $app['config']->get('laravel-newrelic::name_provider');
```

Comment:
@tomschlick I'm not familiar with laravel 5's syntax for config, but I noticed that all the other settings like this one got changed from `::` to `.` as a delimiter. Should this be updated here?